### DISCUSSION TITLE
Experiencing lag  &cmd install

### DISCUSSION THREAD

**Discussion by laneydawf** [rustdesk/rustdesk#9901] [Q&A]
To ensure smooth usage, what configuration is required? I have a server with  2c 2g 1m , but it’s very slow when devices are not on the same network.

When I use the CMD script from this webpage(https://rustdesk.com/docs/en/self-host/client-deployment/) to install the software, it can't import the configuration information.Whether or not the script is run as an administrator, it can only install the software but cannot import the configuration file.
![image](https://github.com/user-attachments/assets/80c3ce44-7caf-40c8-9eab-5007d3e486ef)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/763, go to [discord](https://discord.gg/nDceKgxnkV) please, search there first, some users might have asked similar questions. Usually nobody will help you with build or self-hosting here.
